content/docs/integrations/home-page-markdown/index.md

```diff
@@ -1,25 +1,36 @@
 ---
-title: Home Page Markdown Plugin
+title: Markdown Card Plugin
 publishedDate: '2022-03-01T10:00:00.0Z'
-description: How to enable the home page markdown plugin
+description: How to enable the markdown card plugin
 
-humanName: Home Page Markdown
+humanName: Markdown Card Plugin
 logoImage: '../../../assets/logos/home-page/home.png'
 integrationType: OSS plugin
 ---
 
 ## Introduction
 
-This is a plugin that renders a markdown file from github as a homepage componenet.
+Roadie provides two different versions of Markdown Card plugins that can be used to display content on either the [Home page](/docs/integrations/home-age) or [Catalog dashboard pages](/docs/details/updating-the-ui/#updating-dashboards). 
 
-It fetches the .md file from github on every load and caches it's content.
-To be able to see the rendered content make sure you are logged into github.
 
-## Setup
+#### OAuth Home Page Markdown Card
 
-To use this plugin make sure you already have the [Backstage Home plugin](https://github.com/backstage/backstage/blob/master/plugins/home/README.md) enabled.
+This is a plugin that renders a markdown file from GitHub as a homepage component.
 
-### Add Markdown card to your homepage
+It fetches the .md file from GitHub on every load and caches it's content.
+To be able to see the rendered content make sure you are logged into GitHub.
+
+#### Roadie Markdown Card
+
+Roadie Markdown Card provides possibility to retrieve and display markdown content from sources that you have configured integrations for on your Roadie instance. To see available integrations, take a look at the integrations section on the [plugins page](/docs/integrations/). 
```

Comment:
This sentence is missing a word or two. "The Roadie Markdown Card provides the possibilty...", also the end of the sentence "from content provided by the integrations you have configured."